### PR TITLE
Ports: Prefix output with the build step and port name

### PR DESCRIPTION
**Pull request by gunnarbeutner**
We already have something similar for the toolchain builds. This makes it easier to identify which build step is currently running.

Screenshot from 2022-10-23 15-59-42

### CODE REVIEW

File: Ports/.port_include.sh [hunk 1]
```diff
@@ -10,6 +10,20 @@ unset SERENITY_STRIPPED_ENV
 
 export MAKEJOBS="${MAKEJOBS:-$(nproc)}"
 
+buildstep() {
+    NAME=$1
+    shift
+    if [ -z "$@" ]; then
```
Comment (linusg) [inline Ports/.port_include.sh:16]: Who invented this and thought "yes, that's readable code"?!

**Review by linusg — APPROVED** [review]

**Comment by demostanis** [comment]
Looking at the code made me wonder why some variable names are uppercase while some others are lowercase. Is there a specific coding style rule for shell scripts?

File: Ports/.port_include.sh (hunk 1 above)
Comment (linusg) [reply Ports/.port_include.sh:16]: `./Meta/serenity.sh run lagom Shell` :wink:

**Comment by gunnarbeutner** [comment]
Personally I prefer lower-case for "local" variables. In general our scripts aren't really following any particular style - at least, not as far as I can see.

I've changed `$NAME` to `$buildstep_name` now though.

**Pull request merged**